staging/src/k8s.io/apiserver/pkg/server/lifecycle_signals.go

```diff
@@ -188,3 +196,11 @@ func (e *namedChannelWrapper) Signaled() <-chan struct{} {
 func (e *namedChannelWrapper) Name() string {
 	return e.name
 }
+
+func (e *namedChannelWrapper) SignaledAt() time.Time {
+	value := e.signaledAt.Load()
+	if value == nil {
+		return time.Time{}
```

Comment:
why zero value instead of simply nil?

Reply:
I don't think we need to distinguish between nil and empty value here, but i made it `SignaledAt() (time.Time, bool)`